provisioning/wildfly_channel_in_zips.adoc

```diff
@@ -0,0 +1,210 @@
+* https://github.com/wildfly-extras/prospero[Prospero]
+** Prospero will be used as the library to update WildFly installations but will not surface to the user (the `installation-manager` script is the user entry point)
+* https://github.com/wildfly/wildfly-maven-plugin[wildfly-maven-plugin]
+** The WildFly Maven Plug-in provides the functionality to provision WildFly to create the distributions with the expected channel metadata.
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+* [x] Traditional standalone server (unzipped)
+
+== Requirements
+
+* WildFly distributions archives (zip & tgz) must contain the channel metadata that describes their installation state.
+** tracked by https://issues.redhat.com/browse/WFLY-19221[WFLY-19221]
+** Provisioned feature packs remain referenced by their Galleon feature-pack-locations (e.g. `wildfly@maven(org.jboss.universe:community-universe):current#33.0.0.Final`).
+*** Users that use Galleon tool to udpate their WildFly installation will continue to be able to do so.
+*** Galleon universe artifacts (`org.jboss.universe:community-universe` & `org.jboss.universe.producer:wildfly-producers`) will be added to the WildFly channels
+* WildFly distributions archives (zip & tgz) must contain the tooling to let users update their installations
+** tracked in https://github.com/wildfly-extras/prospero/issues/749[prospero #749]
+** Users must be able to list and select the updates to apply to their installations
+** The user MUST specify the updates to apply.
+** As this feature is experimental, the tooling should warn the user that updating their installation is an experimental mechanism
+** these operations will be using Prospero that needs to be integrated as a JBoss module in the WildFly distributions.
+* Updates must not discard any user changes to an installation (in their configuration files or JBoss modules directory)
+
+=== Non-Requirements
+
+* Changing the type of distributions during an update is not supported (in other words, it is not possible to download the zip for WildFly 33.0.0.Final and update the installation to WildFly Preview)
+* Trimming an existing installation coming from WildFly distributions with Galleon layers is not supported.
+
+=== Future Work
+
+This feature is `experimental`.
+
+To make it a `preview` or `community` feature, we will pay attention to the user experience. In particular, the distributions should be "self-updatable" and should not need
+to rely on the external download of Prospero to be updated. Promotion to `preview` or `community` would involve the integration of Prospero library into WildFly (as a JBoss module or a separate feature pack). The user interface could also be directly the `prospero` cli tool, a streamlined CLI focused on updates, or additional commands provided within `jboss-cli` tool.
+
+=== Implementation Plan
```

Comment:
Should the implementation plan mention the new module(s) created to include Prospero libraries?